coreos-devel/fero-client/fero-client-9999.ebuild

```diff
@@ -23,14 +23,25 @@ SLOT="0"
 
 IUSE=""
 
-DEPEND=">=dev-libs/protobuf-3.0.0"
-RDEPEND="${DEPEND}"
+DEPEND=">=dev-libs/protobuf-3.0.0
+        dev-util/cmake"
+RDEPEND=">=dev-libs/protobuf-3.0.0"
 
 src_unpack() {
 	cros-workon_src_unpack "$@"
 	cargo_src_unpack "$@"
 }
 
+src_prepare() {
+	# grpc-sys-0.2.3 crate build breaks with glibc-2.30 and above (see patch)
```

Comment:
Is this "crate" here a typo for create? Or is it a noun and I'm missing some context?

Regardless of that, I'm a bit confused as to why this is patching grpc-sys in the fero-client package. This seems wrong... Why not patch grpc-sys?

Reply:
Crates are a Rust term for libraries on the official library hub. One could try to patch it on the Cargo level but at that point it's even easier to update the dependency since we would already touch the fero repo.

Reply:
"crate" is a [rust crate](https://crates.io/), i.e. a package. We don't include grpc-sys anywhere, it's downloaded from the Rust crates repo during fero-client's build. I'll elaborate the comment in src_prepare.